shared/people/item/index.js

```diff
@@ -70,7 +70,7 @@ export default (props: Props) => (
       {props.badged && (
         <Badge
           badgeNumber={null}
-          badgeStyle={{marginLeft: globalMargins.tiny, height: 10, width: 10, minWidth: 10}}
+          badgeStyle={{marginLeft: globalMargins.tiny, height: 8, width: 8, minWidth: 10}}
```

Comment:
`minWidth: 10 -> 8`

Reply:
Yeah it was actually 8, not 10.

Reply:
Yeah, I mean `minWidth` should match `width` here or else the desktop badge looks a little stretched:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/11968340/35069877-7cd66766-fba9-11e7-93e9-3e60440d7cd3.png">

Reply:
Ha! Ok updated.